.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "0.2.0",
```

Comment:
How does the versioning work? Is this value arbitrary?

Reply:
I'm not sure tbh, I think it's internal versioning for VSCode launch schema. See in their source code here: https://github.com/microsoft/vscode/blob/4f0c8af89496497d57982f748901c94e75436f43/src/vs/workbench/contrib/debug/common/debugSchemas.ts#L197.